examples/react-apollo/cypress/e2e/1-unauthenticated/forgot-password.cy.ts

```diff
@@ -7,6 +7,7 @@ context('Forgot password', () => {
     cy.signUpAndConfirmEmail(email)
     cy.signOut()
 
+    // cy.visit('/sign-in')
```

Comment:
```suggestion
```

Reply:
Sorry, I was too eager to comment on this PR. Just noticed that this is still a draft.